src/ceph_mgr.cc

```diff
@@ -43,18 +43,18 @@ int main(int argc, const char **argv)
 {
   ceph_pthread_setname(pthread_self(), "ceph-mgr");
 
-  vector<const char*> args;
+  std::vector<const char*> args;
```

Comment:
(\todo: create a 2'nd args_to_vec() with the following signature:
std::optional<std::vector<const char*>> argv_vec(argc, argv)

Cleaner, and will be able to allocate the correct size for the vector.

Reply:
this is not part of this change.